### PR TITLE
Update CelOS logo

### PR DESCRIPTION
**Pull request by C47F15H**

### CODE REVIEW

**Comment by C47F15H** [comment]
#1804

File: neofetch [hunk 1]
```diff
@@ -6533,22 +6533,28 @@ EOF
...
+    ${c3}#################################### ${c4}MMMMMMMMMMMMMc
+          "MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM:
+           "MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM"
+           'MMMMMMMMM*******************************:
+            \"MMMMMM ${c3}#####################################
+             ${c4}`:MMMMMMmmmmmmmmmmmmmmmmmmmmmmmmmmmmm;
+               `"MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM"
+                 `":MMMMMMMMMMMMMMMMMMMMMMMMM;'
+                    `":MMMMMMMMMMMMMMMMMMM:"
+                         "************"
```
Comment (dylanaraps) [inline neofetch:6557]: Remove all space on the left of the logo. (See how others are formatted).

**Comment by dylanaraps** [comment]
Your pull request has unrelated changes.

**Comment by C47F15H** [comment]
> Your pull request has unrelated changes.

For some reason, the spaces in other logos have been removed

**Pull request closed without merging**